CMakeLists.txt

```diff
@@ -2,14 +2,16 @@ cmake_minimum_required(VERSION 3.14)
 
 project(n_e_s)
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-include(sanitizers)
+include(cmake/sanitizers.cmake)
+include(third_party/gtest.cmake)
```

Comment:
Like this, or one CMakeLists.txt in third_party with everything?

Reply:
I was thinking more something like
```
third_party/gtest/CMakeLists.txt
third_party/fmt/CMakeLists.txt
```
but this is fine too!

Reply:
Sure, I fixed that now :)